src-docs/src/views/button/button_example.js

```diff
@@ -324,46 +323,42 @@ export const ButtonExample = {
       ],
       text: (
         <>
-          <EuiCallOut
-            color="warning"
-            title={
-              <span>
-                <strong>EuiButtonToggle</strong> has drastically reduced in
-                functionality. See below for details.
-              </span>
-            }
-          />
-          <EuiSpacer size="m" />
           <p>
             You can create a toggle style button with any button type like the
             standard <strong>EuiButton</strong>, <strong>EuiButtonEmpty</strong>
             , or <strong>EuiButtonIcon</strong>. Use state management to handle
             the visual differences for on and off. Though there are two
```

Comment:
I think you need the `{' '}` thing here. Seems to be missing the space in the rendered output. 
```suggestion
            the visual differences for on and off. Though there are two{' '}
```